hedera-node/hedera-evm-api/src/main/java/com/hedera/services/evm/contracts/operations/HederaBalanceOperation.java

```diff
@@ -46,9 +46,8 @@
 
 /**
  * Hedera adapted version of the {@link BalanceOperation}. Performs an existence check on the
- * requested {@link Address} Halts the execution of the EVM transaction with {@link
- * HederaExceptionalHaltReason#INVALID_SOLIDITY_ADDRESS} if the account does not exist or it is
- * deleted.
+ * requested {@link Address} Halts the execution of the EVM transaction with if the account does not
```

Comment:
Why 
`{@link * HederaExceptionalHaltReason#INVALID_SOLIDITY_ADDRESS} ` is missing?